clipper_admin/clipper_manager.py

```diff
@@ -58,20 +58,32 @@
 EXTERNALLY_MANAGED_MODEL = "EXTERNAL"
 
 
+from numpy import *
+import scipy as sp
```

Comment:
Do you need to import numpy and scipy here? If you do, don't change the numpy import to `import numpy as np` instead of importing `*`.